x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -239,4 +242,81 @@ public static void assertRequestBreakerEmpty() throws Exception {
             }
         });
     }
+
+    public static class ClusterWithCsvData implements TestRule {
```

Comment:
Can you extend the `Cluster` rule we already have? Or include it as a member or something?

Reply:
Thanks Nik, I planned to move some things around and clean that up - I'll try your suggestion as part of that!